### PR TITLE
Fix wrong list URL example

### PR DESCRIPTION
**Pull request by DjebbZ**

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -850,7 +850,7 @@ For instance, let's imagine you have to use the `my.api.url` REST API, which exp
 
 | Action              | Expected API request                                                                    |
 | ------------------- | --------------------------------------------------------------------------------------- |
-| Get list            | `GET http://my.api.url/posts?sort=["title","ASC"]&range=[0, 24]&filter={"title":"bar"}` |
+| Get list            | `GET http://my.api.url/posts?sort=title&order=ASC&page=1&perPage=50&filter={"title":"bar"}` |
```
Comment (fzaninotto) [inline docs/Tutorial.md:853]: No, this doesn't match the dataProvider example below.

Comment (DjebbZ) [reply docs/Tutorial.md]: Not sure what's wrong. Is it the number of items per page? (See next commit)

Comment (djhi) [reply docs/Tutorial.md]: Check the example dataProvider for getList starting at L876

Comment (DjebbZ) [reply docs/Tutorial.md]: Oh I see I've mistaken the URL of the list view page in the React-admin app (which should approx. look like what I'm proposing) with the URL of the call to the API. My bad :sweat_smile:

**Pull request closed without merging**